Sources/GoTrue/Types.swift

```diff
@@ -526,3 +535,43 @@ public struct RecoverParams: Codable, Hashable, Sendable {
     case gotrueMetaSecurity = "gotrue_meta_security"
   }
 }
+
+public struct PKCEParams: Codable, Hashable, Sendable {
```

Comment:
I'd rename this for consistency.

```suggestion
public struct PKCECredentials Codable, Hashable, Sendable {
```